www/front_src/src/Resources/Details/Body/tabs/Details/index.tsx

```diff
@@ -69,19 +69,19 @@ const DetailsTab = ({ details }: Props): JSX.Element => {
   }
 
   const copyCommandLine = (): void => {
-    writeToClipboard(details.command_line as string)
-      .then(() => {
-        showMessage({
-          message: labelCommandCopied,
-          severity: Severity.success,
-        });
-      })
-      .catch(() => {
-        showMessage({
-          message: labelSomethingWentWrong,
-          severity: Severity.error,
-        });
+    try {
+      copy(details.command_line);
+
+      showMessage({
+        message: labelCommandCopied,
+        severity: Severity.success,
+      });
+    } catch (_) {
```

Comment:
is underscore needed in coding style rules ?

Reply:
Yes, the empty catch block is not allowed